### PR TITLE
feat: update alerts filter

### PR DESCRIPTION
**Pull request by neringaalt**
Adding status, closed and startTime fields for AlertFilter

### CODE REVIEW

File: packages/beta/src/types.ts [hunk 1]
```diff
@@ -145,6 +158,9 @@ export interface MonitoringTaskFilterQuery extends FilterQuery {
 export interface AlertFilter {
   channelIds?: CogniteInternalId[];
   channelExternalIds?: CogniteExternalId[];
+  closed?: boolean;
+  startTime?: StartTime;
```
Comment (polomani) [inline packages/beta/src/types.ts]:
```suggestion
  startTime?: DateRange;
```

File: packages/beta/src/types.ts [hunk 2]
```diff
@@ -57,6 +57,19 @@ export const MonitoringTaskModelExternalId = {
...
+  FIRING: 'FIRING' as AlertStatus,
+  PENDING: 'PENDING' as AlertStatus,
+};
+
+export interface StartTime {
+  min?: number;
+  max?: number;
+
+};
+
```
Comment (polomani) [inline packages/beta/src/types.ts]: no need of this type, use DateRange;
```suggestion
```

Comment (neringaalt) [reply packages/beta/src/types.ts]: ok, thanks! :D just trying my best here :D

**Review by tuanng-cognite — APPROVED** [review]
Let's ship this

**Pull request merged**